src/main/java/com/codejoust/main/model/Account.java

```diff
@@ -52,12 +52,16 @@
     private List<ProblemTag> problemTags = new ArrayList<>();
 
     // List of tags associated with this problem
-    @ManyToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE, CascadeType.REFRESH, CascadeType.DETACH}, fetch = FetchType.EAGER)
+    @ManyToMany(fetch = FetchType.EAGER, cascade = CascadeType.ALL)
     @Setter(AccessLevel.PRIVATE)
     @JoinColumn(name = "game_report_id")
     @Fetch(value = FetchMode.SUBSELECT)
     private List<GameReport> gameReports = new ArrayList<>();
 
     @Enumerated(EnumType.STRING)
     private AccountRole role = AccountRole.TEACHER;
+
+    public void addGameReport(GameReport gameReport) {
+        gameReports.add(0, gameReport);
```

Comment:
That's a good question - it might not make a difference, I haven't seen a different ordering system pop up under Sequel Pro. If that's the case, I may revert this to `gameReports.add(gameReport);` in the future for simplicity, but I'll check this when I make the relevant frontend PR.